lib/bundle/brew_installer.rb

```diff
@@ -9,11 +9,18 @@ def initialize(name, options = {})
         @full_name = name
         @name = name.split("/").last
         @args = options.fetch(:args, []).map { |arg| "--#{arg}" }
+        @restart = options.fetch(:restart_service, false)
       else
         raise "Unable to install #{name}. Homebrew is not currently installed on your system"
       end
     end
 
+    def run
+      install_or_upgrade && restart
```

Comment:
I don't think we'll want to have `sudo` support in this DSL and I don't think this method needs to change name to add optional support for restarting a service. Sorry!